Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mambaorg/micromamba:2.1.1
```

Comment:
The latest version is 2.3.2, why not use the latest one?

Reply:
This was based on the Dockerfile I submitted into the gitlab repo a while ago. I've updated it now.